ui/components/ui/icon-button/icon-button.scss

```diff
@@ -29,4 +29,8 @@
     opacity: 0.3;
     cursor: auto;
   }
+
+  p {
+    width: 60px; //for ellipsis keeping the width same as icon-button used here
```

Comment:
Should this selector be `.icon-button__label` instead of `p`?

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/19920/commits/abdb605a862c80656a4ebd45616e5e8168096973